templates/includes/sidebar.html

```diff
@@ -1,4 +1,17 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+     <li>User: <a href="{% url "taxi:driver-detail" pk=user.id %}"> {{ user.get_username }}</a></li>
```

Comment:
The method `user.get_username` should be called with parentheses as `user.get_username()`. This is a method call, and omitting the parentheses will result in a template error.